prawns/balances/db/migrations/1649868972554-Data.js

```diff
@@ -0,0 +1,15 @@
+module.exports = class Data1649868972554 {
```

Comment:
there's only supposed the be 1 migrations file (they're not really migrations). Please delete them all and run yarn migration again

Reply:
The devkit cli tool has a command to do it for you.

`devkit scaffolding balances`

Reply:
Probably needs a better name than scaffolding ... maybe schema:create

Reply:
There's something I did in the EVM work that handles this for you, I've removed it now I've moved away from postgres and don't need migrations though. We can gitignore all the models, types & migrations code which forces a fresh build//scaffold of everything. Makes the PRs a lot nicer too. When we take on either governance or container orchestration in the squid I think it should be preceded with a little tech debt cleanup in the prawns workflow, which will have a knock on effect in the cli package

Reply:
Thanks guys. I'll regenerate this. 

What's the underlying reason why we only have one migration? I know we're not migrating as such (right now), does this just make things simpler and/or help avoid conflicts somehow?